### DISCUSSION TITLE
I have deployed the SPI but can not see the user

### DISCUSSION THREAD

**Discussion by dwina** [dasniko/keycloak-extensions-demo#2] [General]
I have deployed the SPI and selected and filled like in video. Can you post the authorisation app too, where the base.url can verify the user and password. Because without that I can not listen the user in the users category

**Comment by dasniko** [comment]
First of all, "the SPI" is misleading, as this repository contains various SPI implementations. 
But from your description I guess, you mean the User Storage SPI...

This repository is meant as a place for code examples. External services, like mentioned in one of my videos, are not guaranteed to be available for public access. So is the Peanuts API, it's a private API of me, no more available under the URL used in the video.
